mimic/model/heat_objects.py

```diff
@@ -135,6 +135,8 @@ def creation_response_json(self, absolutize_url):
 
 
 stack_creation = EventDescription()
+stack_check = EventDescription()
```

Comment:
Oh, I didn't mean that they shouldn't be used or are not useful, just that they are not needed here yet.